getting-started-tutorial/tutorial-2-Model-training.ipynb

```diff
@@ -0,0 +1,2021 @@
+{
```

Comment:
<p>In the previous tutorial you discussed only about <code># nuclio: ignore</code>  while actually only using <code># nuclio: start-code</code>  / <code># nuclio: end-code</code>  .</p><p>Maybe introduce it back then?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>